### PR TITLE
fix

### PR DESCRIPTION
**Pull request by anis3**

### CODE REVIEW

File: lib/resolveurl/plugins/anistream.py [hunk 1]
```diff
@@ -1,24 +1,44 @@
...
     pattern = r'(?://|\.)(ani-stream\.com)/(?:embed-)?([0-9a-zA-Z-]+)'
+
+
+    def get_media_url(self, host, media_id):
+        web_url = self.get_url(host, media_id)
+        headers = {'User-Agent': common.FF_USER_AGENT}
+        html = self.net.http_GET(web_url, headers=headers).content
+        r = re.search(r'base64,([^"]+)',html)
+        if r:
+            html = base64.b64decode(r.group(1)).decode('utf-8')
```
Comment (Gujal00) [inline lib/resolveurl/plugins/anistream.py:38]: Please test this in Kodi 19 as the current code will cause issue in Python3
specifically the decode and base64 have to be handled correctly for Python3

File: lib/resolveurl/plugins/anistream.py [hunk 2]
```diff
@@ -1,24 +1,44 @@
...
 
-You should have received a copy of the GNU General Public License
-along with this program. If not, see <http://www.gnu.org/licenses/>.
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
-from resolveurl.plugins.__resolve_generic__ import ResolveGeneric
 
+from resolveurl.plugins.lib import helpers
+import re, base64
```
Comment (Gujal00) [inline lib/resolveurl/plugins/anistream.py]: Please follow FLAKE8 standards, one import per line

File: lib/resolveurl/plugins/anistream.py [hunk 3]
```diff
@@ -1,24 +1,44 @@
 """
-ani-stream resolveurl plugin
-Copyright (C) 2016 quartoxuna
+    Plugin for ResolveUrl
+    Copyright (C) 2020 gujal
```
Comment (Gujal00) [inline lib/resolveurl/plugins/anistream.py]: you can add your name for the copyright :-)

**Review by Gujal00 — CHANGES_REQUESTED** [review]

**Comment by anis3** [comment]
I have tested it with kodi 18 (win10 & android) & 19(win10) works

**Review by Gujal00 — APPROVED** [review]

**Pull request merged**